app/views/admin/administrators/administrators/index.html.erb

```diff
@@ -1,22 +1,18 @@
-<div class="govuk-grid-row">
-  <div class="govuk-grid-column-two-thirds">
```

Comment:
Do we not want it to be two-thirds? Just making sure.

Reply:
I'd say that table-based pages like this should probably be full width

Reply:
The other table page is full width and looks better, just changing this one for consistency